src/text-editor-element.coffee

```diff
@@ -153,8 +153,10 @@ class TextEditorElement extends HTMLElement
     @component.focused() if event.relatedTarget is this
 
   addGrammarScopeAttribute: ->
```

Comment:
The only reason I can see it needing to be in the view would be to support the previous command-based approach, since commands are tied to the view.